tools/payloads/configbytes/apk_configbytes_extract.rb

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env ruby
+# -*- coding: binary -*-
+
+class ExtractError < StandardError; end
+class HelpError < StandardError; end
+class UsageError < ExtractError; end
+
+require 'rex'
+require 'tmpdir'
+require 'optparse'
+require 'open3'
+require 'nokogiri'
+require 'fileutils'
+
+def run_cmd(bin, args, verbose)
+  begin
+    path = Rex::FileUtils.find_full_path(bin)
+    cmd = [path, args].join(" ")
+    stdin, stdout, stderr = Open3.popen3(cmd)
+
+    if verbose
+      $stdout.puts "[+] Running: #{cmd}"
```

Comment:
A running message like this should really be printed before the command is run. If `Open3` fails for whatever reason, the running message won't be displayed and it's value will be negated.